src/gamegrid.js

```diff
@@ -0,0 +1,9 @@
+let rows = 20;
+let cols = 10;
+let gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
```

Comment:
As these aren't reassigned it'd be better to mark these as const.

Note that you can edit an item in an array while it's const, you just can't reassign a whole new array in its place
```suggestion
const rows = 20;
const cols = 10;
const gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
```

Reply:
change has been made. thanks